packages/uikit-workshop/gulpfile.js

```diff
@@ -71,6 +77,13 @@ gulp.task('build:css', function() {
         { map: false }
       )
     )
+    .pipe(
+      cleanCSS({
```

Comment:
I think you could argue that removing whitespace from the output reduces comprehension. @bradfrost do you have thoughts on this? See the resulting diff: https://github.com/pattern-lab/patternlab-node/pull/904/files#diff-c44923a0878fc1c446276a7c21a7666d

Reply:
Source maps can definitely help with the readability of the bundled assets shipping in PL's UIKit (without taking a big performance hit in the process).

I'd personally recommend two things:
1. That we include a `--prod` config flag that the build checks against when compiling so we can get the best of both worlds: easier to read code when doing dev work (esp. local dev work), can shave off about `28.4KB` of CSS when consumers are shipping / using PL's UI in production + enough flexibility to keep everyone happy

2. We remove precompiled assets like the `dist` folder from Git so PL users can more easily customize when consuming (I'm still in support of including this `dist` folder when publishing to NPM though out of convenience!)

Reply:
^ We'd also get the added bonus of fewer dist-folder specific merge conflicts....
![image](https://user-images.githubusercontent.com/1617209/43336509-26a21f84-919f-11e8-8ef7-984400a338b6.png)

Reply:
Aye. the inclusion was definitely born out of a need for direct npm install without having to invest in a lifecycle hook. I think I am more comfortable with how to do that now, so a `prepublish` step of some sort I think would be great